lib/Command/Export.php

```diff
@@ -42,19 +41,11 @@
 use Symfony\Component\Console\Style\SymfonyStyle;
 
 class Export extends Base {
-	private IUserManager $userManager;
-	private UserMigrationService $migrationService;
-	private ITempManager $tempManager;
-
 	public function __construct(
-		IUserManager $userManager,
-		UserMigrationService $migrationService,
-		ITempManager $tempManager
+		private IUserManager $userManager,
```

Comment:
**issue:** This PHP 8.0+ syntax will not work with PHP7.4, which is the oldest version supported with NC 25.